src/app/loadout-builder/loadout-builder-reducer.ts

```diff
@@ -114,21 +114,13 @@ const lbStateInit = ({
           if (item && isLoadoutBuilderItem(item)) {
             pinnedItems[item.bucket.hash] = item;
           } else if (item && item.bucket.hash === BucketHashes.Subclass && item.sockets) {
-            const abilityAndSuperSockets = getSocketsByCategoryHashes(item.sockets, [
-              SocketCategoryHashes.Abilities_Abilities_DarkSubclass,
-              SocketCategoryHashes.Abilities_Abilities_LightSubclass,
-              SocketCategoryHashes.Super,
-            ]);
-            const socketOverridesForLO = { ...loadoutItem.socketOverrides };
-
             // In LO we populate the default ability plugs because in game you cannot unselect all abilities.
-            for (const socket of abilityAndSuperSockets) {
-              if (!socketOverridesForLO[socket.socketIndex]) {
-                socketOverridesForLO[socket.socketIndex] =
-                  socket.socketDefinition.singleInitialItemHash;
-              }
-            }
-            subclass = { ...item, socketOverrides: loadoutItem.socketOverrides };
```

Comment:
The unused `socketOverridesForLO` seemed like a bug?

Reply:
Yeah, interesting that it didn't cause issues. From what I can see it should mean default socket overrides don't get set. We are probably checking that in a few areas which means that it doesn't matter it gets missed here.